maximum-cut/maximum_cut.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 D-Wave Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## ------- Set up our graph -------
+import networkx as nx
+
+# Create empty graph
+G = nx.Graph()
+
+# Add edges to the graph (also adds nodes)
+G.add_edges_from([(1,2),(1,3),(2,4),(3,4),(3,5),(4,5)])
+
+## ------- Set up our QUBO dictionary -------
+
+# Initialize our Q matrix with all zeros
+Q = {}
+for u in G.nodes:
+    for v in G.nodes:
+        Q[(u,v)]=0
+
+# Update Q matrix for every edge in the graph
+for uv in G.edges:
+    u = uv[0]
+    v = uv[1]
+    Q[(u,u)]+= -1
+    Q[(v,v)]+= -1
+    Q[(u,v)]+= 2
+
+## ------- Run our QUBO on the QPU -------
+# Set up QPU parameters
+chainstrength = 8
+numruns = 10
+
+# Run the QUBO on the solver from your config file
+from dwave.system.samplers import DWaveSampler
+from dwave.system.composites import EmbeddingComposite
+response = EmbeddingComposite(DWaveSampler()).sample_qubo(Q, chain_strength=chainstrength, num_reads=numruns)
+energies = iter(response.data())
+
+## ------- Return results to user -------
+print('-' * 60)
+print('{:>15s}{:>15s}{:^15s}{:^15s}'.format('Set 0','Set 1','Energy','Cut Size'))
+print('-' * 60)
+for line in response:
+    S0 = list([k for k,v in line.items() if v == 0])
```

Comment:
There seems to be two lists here. Either use `list(..)` or the square brackets `[..]`, but not both.
Namely,
```
option1 = list(k for k,v in line.items() if v == 0)
option2 = [k for k,v in line.items() if v == 0]
```

Not relevant to the PR, but still interesting (imho). When thinking about performance, the choice of `option1` and `option2` can matter. `option1` generates the values on demand (and helps to save memory usage), while `option2` constructs the entire list.  https://wiki.python.org/moin/Generators

Reply:
@m3ller, second paragraph not true. Two options are identical in time and memory performance. What you probably meant was:
```
option1 = (k for k,v in line.items() if v == 0)
option2 = [k for k,v in line.items() if v == 0]
```
Now `option1` is a generator, and `option2` is a list.

Reply:
Yup, you're right. Thanks for catching that, @randomir!